src/modules/currency/__tests__/formatAmount.test.ts

```diff
@@ -0,0 +1,44 @@
+import formatAmount from "../formatAmount";
+
+describe("formatAmount", () => {
+  it("should format the amount with default options", () => {
+    const result = formatAmount("USD", "1000.5");
+    expect(result).toBe("$1,000.50");
+  });
+
+  it("should format the amount with custom locale and currency display", () => {
+    const result = formatAmount("EUR", "1500", {
+      locale: "fr-FR",
+      currencyDisplay: "code",
+    });
+    expect(result).toBe("1 500,00 EUR");
+  });
+
+  it("should format the amount without currency symbol", () => {
+    const result = formatAmount("GBP", "750.75", { withSymbol: false });
```

Comment:
jestdom does not have a default value for the locale, and so fallback is what will be taken as locale here which is 'en-US'. 

This UT won't produce different output for different geography since locale doesn't changes as it considers fallback value which will remain same irrespective of what geography user is in.